bootstrap-with-conda-env.bat

```diff
@@ -0,0 +1,3 @@
+yarn conda:create &&^
+conda activate pmp &&^
```

Comment:
do you need `call conda activate pmp` here?

Also, what does the `&&^` mean?

(only if you think you need it...)
```suggestion
call conda activate pmp &&^
```

Reply:
- `&&` only executes the next command if the previous command executed successfully (status code 0, I believe)
- `^` is a line continuation flag.
- As I understand it, `call` kinda achieves the opposite of `&&`, the next command will run regardless of the success of the current command...

Reply:
I believe `call` says, run that `.bat` file then come back here with those changes made. I think it is needed for `conda activate` within a bat file...

It's so funny (frustrating) that `&&` actually needs to be written and isn't the default behavior.

`^` line continuation?